cves/CVE-2016-7905.yml

```diff
@@ -352,7 +383,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The research that was done, I have concluded that thee notable mistakes
+    could lead these mistakes to stay unnoticed for this long, it that with
+    multiple developers including a few new developers maintainability
+    miscommunication with the lack of testing allowed this vulnerability to
+    be unnoticed. There was no testing found that shows that the function was
+    thoroughly tested nor tested after finding the vulnerability.
```

Comment:
I would suggest re-structure the first sentence by splitting into couple sentences - it is hard to read/ understand.